cxplat/inc/cxplat/common.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+#include <assert.h>
+#include <sal.h>
+
+#ifdef CXPLAT_SOURCE
+#define CXPLAT_API __declspec(dllexport)
+#else
+#define CXPLAT_API __declspec(dllimport)
+#endif
+
+#ifdef _DEBUG
```

Comment:
FYI, I just found out that `_DEBUG` generally isn't universally defined for debug mode. We can discuss this more later, but we had to go the route of:
```c
#ifndef NDEBUG
#define DBG 1
#define DEBUG 1
#endif
```
And NDEBUG is defined for release builds.

Reply:
Updated